code/processes/dqe.q

```diff
@@ -15,7 +15,7 @@ resultstab:([]procs:`$();funct:`$();table:`$();column:`$();resvalue:`long$());
 init:{
   .lg.o[`init;"searching for servers"];
   /- Open connection to discovery
-  .servers.startup[];
+  .servers.startupdependent[`dqedb;10];
```

Comment:
Why this change?

Reply:
Due to this:

```
.servers.CONNECTIONS:`tickerplant`rdb`hdb`dqedb
```

in the previous comment. If it wasn't startupdependent, then it didn't start up properly.